claymore-frontend/src/pages/Employee.tsx

```diff
@@ -0,0 +1,167 @@
+import Header from "../components/Header";
```

Comment:
Can just remove this page as well based on the prior comment.

Reply:
I think we need a page to see customer orders so we know who ordered what. So I’m keeping this for now.